pegasus/migrations/76_migrate_channel_sources_to_s3.rb

```diff
@@ -0,0 +1,60 @@
+require 'cdo/hip_chat'
+require 'files_api'
+
+# Migration for moving JSON.parse(row[:value])['levelSource'] and ['levelHtml'] to S3 (/v3/sources API).
+Sequel.migration do
+  up do
+    HipChat.log 'Moving <b>channel</b> sources to S3...'
+    FILE_NAME = 'main.json'
+
+    # For each channel with a level(Source|Html) and no migratedToS3 flag, create a new S3 record and set migratedToS3.
+    batch_update do |row|
```

Comment:
Sequel also has [`paged_each`](http://sequel.jeremyevans.net/rdoc/classes/Sequel/Dataset.html#method-i-paged_each), but I kept getting `Sequel::DatabaseDisconnectError: Mysql2::Error: Lost connection to MySQL server during query` when I tried to modify the table I was iterating over.

`batch_update` is just a naive way to operate on a set of records without loading the entire `:storage_apps` table into memory (~700k rows).  I used it previously in https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/migrations/75_add_created_at_column_to_storage_apps.rb#L26.